pandas/core/arrays/timedeltas.py

```diff
@@ -107,8 +112,29 @@ def wrapper(self, other):
     return compat.set_function_name(wrapper, opname, cls)
 
 
+def _wrap_tdi_op(op):
+    """
+    Instead of re-implementing multiplication/division etc operations
+    in the Array class, for now we dispatch to the Timedelta implementations.
```

Comment:
Clarify: TImedelta -> TimedeltaIndex, right?

Reply:
Yes, good catch.  Also needs a big shiny TODO note to implement it directly here and wrap in TimedeltaIndex instead of the other way around.